app/models/planet.py

```diff
@@ -0,0 +1,23 @@
+from app import db
+
+class Planet(db.Model):
```

Comment:
Great addition of instance methods. For your columns, would you make them nullable? Like `nullable = False` .